### ISSUE TITLE
Warnings code

### ISSUE THREAD

**Issue by agustito37** [rootstrap/react-native-use-styles#24]
# Feature Requests

### Checklist

- [x] My feature request is specific to `@rootstrap/react-native-use-styles`.

- [x] I've searched open issues to make sure I'm not opening a duplicate feature request

### Description

Improve warnings code. Today we have a repetitive code to add a development warning; e.g:
```
if (process.env.NODE_ENV !== "production" && someCondition) {
    console.warn(
        `useStyles [warningKey]: "[description]. You are seeing this warning because you are in development mode. In a production build there will be no warning.`
    );
}
```
It would be great to have this logic encapsulated into a function as this:
```
warn(someCondition, description, warningKey);
```

It's important that these warns don't appear in the production build. For that we are using @rollup/plugin-replace, so we need to change its configuration in order to support this.

**Comment by agyin3** [comment]
Can I take this issue?

**Comment by agustito37** [comment]
Yes, of course, go ahead.

**Comment by gajanangitte** [comment]
Hi! I am new to Open Source and this is my 2nd PR. So sorry that I didn't reply earlier (I didn't know that I have to ask to work on an issue 😅). I have PRed to reduce redundant code.

**Comment by agyin3** [comment]
@agustito37 I know @gajanangitte has opened a PR but I would still like to work on it if possible

**Comment by agustito37** [comment]
Ok, no problem. Yes the idea, at least in this repo, is to assign the issue to a person, so people don't waste time working on the same issue. But it's ok, if you both want to provide a PR is fine by me; I check both. Take into consideration I cannot merge both.

**Comment by domuserk** [comment]
Posso analisar outros problemas?

**Comment by agustito37** [comment]
> Posso analisar outros problemas?

You can check the issues if you want.
